listenerutil/parse.go

```diff
@@ -387,3 +413,139 @@ func ParseSingleIPTemplate(ipTmpl string) (string, error) {
 		return "", fmt.Errorf("multiple addresses found (%q), please configure one", out)
 	}
 }
+
+var validCustomStatusCodeCollection = []string{
+	"default",
+	"1xx",
+	"2xx",
+	"3xx",
+	"4xx",
+	"5xx",
+}
+
+const strictTransportSecurity = "max-age=31536000; includeSubDomains"
+const xContentTypeOptions = "nosniff"
+const cacheControl = "no-store"
+const apiContentSecurityPolicy = "default-src 'none'"
+const uiContentSecurityPolicy = "default-src 'none'; script-src 'self'; frame-src 'self'; font-src 'self'; connect-src 'self'; img-src 'self' data:*; style-src 'self'; media-src 'self'; manifest-src 'self'; style-src-attr 'self'; frame-ancestors 'self'"
```

Comment:
Is there a reference for why we chose this as the default CSP?

Reply:
these are the headers that were defined in the ticket

Reply:
I couldn't find a reference in the ticket so w/e I guess 😛